alert-common/src/main/java/com/synopsys/integration/alert/common/event/EventListenerConfigurer.java

```diff
@@ -34,31 +32,28 @@
 import org.springframework.jms.config.JmsListenerEndpointRegistrar;
 import org.springframework.jms.config.SimpleJmsListenerEndpoint;
 
-import com.synopsys.integration.alert.common.descriptor.ChannelDescriptor;
-
 @Configuration
-public class ChannelListener implements JmsListenerConfigurer {
-    private final Logger logger = LoggerFactory.getLogger(ChannelListener.class);
+public class EventListenerConfigurer implements JmsListenerConfigurer {
+    private final Logger logger = LoggerFactory.getLogger(EventListenerConfigurer.class);
 
-    private final List<ChannelDescriptor> channelDescriptorList;
+    private final List<AlertEventListener> alertEventListeners;
 
     @Autowired
-    public ChannelListener(final List<ChannelDescriptor> channelDescriptorList) {
-        this.channelDescriptorList = channelDescriptorList;
+    public EventListenerConfigurer(final List<AlertEventListener> alertEventListeners) {
+        this.alertEventListeners = alertEventListeners;
     }
 
     @Override
     public void configureJmsListeners(final JmsListenerEndpointRegistrar registrar) {
         logger.info("Registering JMS Listeners");
-        channelDescriptorList.forEach(descriptor -> {
-            final MessageListener channelListener = descriptor.getChannelListener();
-            if (channelListener != null) {
-                final String listenerId = createListenerId(descriptor.getName());
+        alertEventListeners.forEach(listener -> {
```

Comment:
Seeing this now, we may want a traditional for-loop here since `.forEach(...)` doesn't buy us anything.